src/app/services/deposits/deposits.service.ts

```diff
@@ -0,0 +1,18 @@
+import { environment } from './../../../environments/environment';
```

Comment:
hey @Genza999, we should use absolute imports

Reply:
Thank @CryceTruly , I have worked on this